tests/modules/services/conky/default.nix

```diff
@@ -1,5 +1,4 @@
-{ lib, pkgs, ... }:
-
-lib.optionalAttrs pkgs.stdenv.hostPlatform.isLinux {
```

Comment:
Don't remove this condition. It's used to prevent us testing on darwin because the module has an assertion for linux.